src/components/button.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/prop-types */
+
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+
+const Button = props => {
+  const ButtonName = props.name;
```

Comment:
use `proptypes` to ensure that `props.name` is a string [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png) ]